hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/NomisService.kt

```diff
@@ -54,5 +54,5 @@ class NomisService(
 
 data class PrisonerNumberPage(
   val content: List<String> = emptyList(),
-  val totalRecords: Long = 0,
+  val totalElements: Long = 0,
```

Comment:
the wiremock test uses this class too hence didn't spot that Prison API is returning something else